test/hemi.tokenlist.test.js

```diff
@@ -89,8 +89,12 @@ describe("List of tokens", function () {
           .replace(".e", "")
           .replaceAll(" ", "-")
           .toLowerCase();
-        assert.equal(logoURI, `${repoUrl}/master/src/logos/${filename}.svg`);
-        fs.accessSync(`src/logos/${filename}.svg`);
+
+        const filepath = logoURI.match(
+          new RegExp(`^${repoUrl}/master/src/logos/${filename}.(svg|png)$`),
+        );
+        assert.notEqual(filepath, null);
+        fs.accessSync(filepath[0].replace(`${repoUrl}/master/`, ""));
```

Comment:
The '.' has to be escaped.

```suggestion
        const filepath = logoURI.match(
          new RegExp(`^${repoUrl}/master/src/logos/${filename}\.(svg|png)$`),
        );
        assert.notEqual(filepath, null);
        fs.accessSync(filepath[0].replace(`${repoUrl}/master/`, ""));
```

The goal of this check was to ensure the logo was included as a file in the repository and to prevent using an incorrect file after a typical copy&paste error.

While the first check is simple and clear, do we want to be that strict with the second check? cc/ @gndelia

Reply:
@gabmontes eslint starts complaining if we escape the `.`

![Captura de Tela 2025-03-28 às 13 11 30](https://github.com/user-attachments/assets/6efba850-d90d-419b-8749-24d8e8922e4a)

Reply:
note that the `\.` was added in a different line, in line 94 (not in the `repoUrl`)

Reply:
@gndelia The alert is related to line 86 as you can see in the following screenshot:

![Captura de Tela 2025-03-28 às 13 40 45](https://github.com/user-attachments/assets/55865ae8-cb6c-41cc-8086-5011ce7fe234)

Reply:
I know I know, it's just that Gabriel suggestion is on line 94, unrelated to the alert 😅

Reply:
@gabmontes @gndelia eslint also complains about escaping `.` on line 94.

![Captura de Tela 2025-03-28 às 13 55 27](https://github.com/user-attachments/assets/7894b517-0c10-402d-9cfb-939cb859001e)

Reply:
then I don't know. 😆  strictly speaking, if we don't escape the point, it could potentially match wrong filenames, allowing any character after `filename` and before the extension to be ignored. I don't think it's required on line 86 (as there's no input from the user), but in this line it is (There's input from the user - us, developers, naming the filename).

So for line 94 I'd disable the eslint rule with `eslint-disable-next-line`

Reply:
eslint disabled on line 94